client/components/costume/AllCostumes.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { loadCostumesDispatch } from '../../redux/costumes/allCostumes';
+
+class AllCostumes extends Component {
+  componentDidMount () {
+    const { loadCostumesDispatch } = this.props;
+    loadCostumesDispatch()
+  };
+  render() {
+    const { 
+      categories, selectedCategoryId, costumes 
+    } = this.props
+    return (
+      <div className="costumesList">
+          <h5>{!selectedCategoryId || isNaN(selectedCategoryId) ? 'All' : 
+          categories.map((category) => category.id === selectedCategoryId ? category.title : null)} Costumes</h5>
+          <div>
+            {!selectedCategoryId || isNaN(selectedCategoryId) ?
+            costumes.map((costume) => (
+              <div key={costume.id} className="costumes">
+                <div>
+                  <Link to={`/costumes/${costume.costumeName}/${costume.id}`}>{costume.costumeName}</Link>
+                </div>
+                <img src={costume.imageUrl} />
+                <div>
+                  <Link to={`/costumes/${costume.costumeName}/${costume.id}/${costume.categoryId}/admin`}>
```

Comment:
Probably not for this round, but given only admins should be able to access this update costume link, I think this would either need a check using userType from user reducer, or just a separate page that has all costume links that only admins can access.

Reply:
totally agree; once we have an authentication workflow in place, these links should only be available to admin-type users